### PR TITLE
Remove snapshot owner filter

### PR DESCRIPTION
**Pull request by makentenza**
##### SUMMARY

Remove one of the filters to search for sap software snapshot

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
sap-hana

### CODE REVIEW

**Comment by makentenza** [comment]
@rut31337 as requested I've removed the filter

File: ansible/configs/sap-hana/pre_infra.yml [hunk 1]
```diff
@@ -32,8 +32,7 @@
     - name: Find out SnapShot ID for SAP Software
       ec2_snapshot_info:
         filters:
-          description: "{{ sap_software_snapshot_name }}"
-          owner-id: "{{ sap_software_snapshot_owner_id }}" 
+          description: "{{ sap_software_snapshot_name }}"
```
Comment (fridim) [inline ansible/configs/sap-hana/pre_infra.yml:35]: There is a security issue about using only the description here.

We need to ensure the origin of the snapshot.

**Review by fridim — CHANGES_REQUESTED** [review]

File: ansible/configs/sap-hana/pre_infra.yml (hunk 1 above)
Comment (makentenza) [reply ansible/configs/sap-hana/pre_infra.yml:35]: @rut31337 please agree with @fridim on this. As long as we know the owner we can add that variable in AgnosticV, I don't see why should not be able to, but let us know what's your point of view here.

**Comment by makentenza** [comment]
Already solved on a different PR

**Pull request closed without merging**